### PR TITLE
README.md: Mention that we will accept GitHub PRs

### PR DESCRIPTION
**Pull request by ocharles**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ There are two ways to get a source tree:
   Note: cloning GHC from Github requires a special setup. See [Getting a GHC
   repository from Github] [7].
 
-  **DO NOT submit pull request directly to the github repo.**
+  We will accept easily-reviewed documentation changes as GitHub Pull Requests.
```
Comment (sjakobi) [inline README.md]: "easily-reviewed" makes sense but I don't think that PRs should be limited to documentation changes up front.

[The proposal](https://github.com/ghc-proposals/ghc-proposals/pull/11) certainly wasn't limited to documentation changes only.

Comment (ocharles) [inline README.md]: I have just taken what's in the GitHub project description. Whatever we write here, the two should agree.

Comment (sjakobi) [inline README.md]: Oh, right, I had overlooked that bit.

Well, on the proposal discussion, quchen mentions ["housekeeping"](https://github.com/ghc-proposals/ghc-proposals/pull/11#issuecomment-269816034) changes that he would like to submit on Github, and which I think should be encouraged too. And if a PR can't easily be handled on GitHub, there's always the option of asking the submitter to move his patch to phabricator.

But let's see what @bgamari or possibly other maintainers are thinking about this.

**Comment by ygale** [comment]
It seems from the discussion in ghc-proposals/ghc-proposals#11 that other small changes will be directly accepted, not just documentation changes.

More importantly, if the GHC team decides that the PR is not small enough to be directly accepted without review, it will not be rejected. Rather, it will be entered into Phab. At least, on a trial basis.

So I think the intent isn't only for small changes - it is to encourage first time contributions. The idea is that more people will get involved if the initial bar is set lower. Once you start contributing, you will naturally start interacting with people via Phab and Trac, and soon you will be contributing in the normal way.

So perhaps it should be something like:

> See the GHC team's working conventions about [how to contribute a patch to GHC](http://ghc.haskell.org/trac/ghc/wiki/WorkingConventions/FixingBugs). First time contributors are encouraged to get started by just sending a PR.

**Comment by bgamari** [comment]
Thanks for picking this up, @ocharles! To clarify, the change does, at least currently, primarily target documentation changes. In general I would really like to avoid having to do code review on two separate platforms. The changes that will be accepted via GitHub are those that I feel don't require any additional review beyond my own two eyes.

> Rather, it will be entered into Phab. At least, on a trial basis.

Right, I am willing to move things to Differential. However, I would prefer not to become a Phabricator-GitHub gateway as the review process proceeds and would ask the submitter to pick up the Differential if their patch does need review. Perhaps some day we will have automation for pulling Git branches into Differentials.

All this being said, I think @ygale's suggested language summarizes nicely,
> See the GHC team's working conventions about how to contribute a patch to GHC. First time contributors are encouraged to get started by just sending a PR.

**Comment by bgamari** [comment]
I have pushed a variant of the text proposed here as afc05c76c3bd672ce04527c89b29d184e94c8c6d.

**Comment by sjakobi** [comment]
> I have pushed a variant of the text proposed here as afc05c7.

That sounds very newcomer-friendly, cheers!

Can you also replace the text in the GitHub repo description with this version?

**Pull request closed without merging**